packages/framework/src/Component/CustomerUploadedFile/Config/CustomerUploadedFileConfigDefinition.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Component\CustomerUploadedFile\Config;
+
+use Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition;
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+class CustomerUploadedFileConfigDefinition implements ConfigurationInterface
```

Comment:
I made the initial changes this way because the product files affected how uploaded files works. At the time, these changes hadn't been merged yet, so I just copied it to prevent conflicts. Now that the product file updates are merged, I've refactored it to abstract same functionality and reuse it across both uploaded files and customer uploaded files.

[abstracted uploaded file and customer uploaded file](https://github.com/shopsys/shopsys/pull/3295/commits/70df98520e870b9edc5104e98e3912f3aa8e337f)

Reply:
awesome!